app/components/projects/index_page_header_component.rb

```diff
@@ -31,14 +31,13 @@
 class Projects::IndexPageHeaderComponent < ApplicationComponent
   include OpPrimer::ComponentHelpers
   include Primer::FetchOrFallbackHelper
+  include Menus::ProjectsHelper
```

Comment:
I couldn't find the reason for including this module. Which method within it is referenced?